cmd/nmi/main.go

```diff
@@ -35,7 +36,7 @@ func main() {
 	if err != nil {
 		log.Fatalf("%+v", err)
 	}
-	s := server.NewServer()
+	s := server.NewServer(*forceNamespaced || "true" == os.Getenv("FORCENAMESPACED"))
```

Comment:
nit: its better to update *forcedNamespaced variable with this check and not doing the check in the NewServer call